fetch.bs

```diff
@@ -1038,6 +1041,77 @@ run these steps:
  <li><p>Return <var>values</var>.
 </ol>
 
+<p>To determine if a <a>byte sequence</a> <var>value</var> is a
+<dfn>simple range header value</dfn>, perform the following steps. They return a <a>boolean</a>.
+
+<ol>
```

Comment:
How about using regexp?

Reply:
I haven't seen regexp used in a spec before (and it raises questions about the flavour of regexp). I'm going to switch to "collect a sequence". Hopefully that'll be more readable.